client/src/pages/donate.js

```diff
@@ -72,21 +85,44 @@ export class DonatePage extends Component {
       <Fragment>
         <Helmet title='Support our nonprofit | freeCodeCamp.org' />
         <Grid>
+          <Spacer />
           <Row>
             <Col sm={10} smOffset={1} xs={12}>
-              <Spacer />
-              <h2 className='text-center'>Become a Supporter</h2>
+              <h1 className='text-center'>Become a Supporter</h1>
               <Spacer />
             </Col>
           </Row>
           <Row>
-            <Col sm={8} smOffset={2} xs={12}>
-              <DonateText />
-              <Spacer />
+            <Col md={6}>
+              <DonateForm
+                enableDonationSettingsPage={this.enableDonationSettingsPage}
+                stripe={stripe}
+              />
+              <Row>
+                <Col sm={10} smOffset={1} xs={12}>
+                  <Spacer size={2} />
+                  <h3 className='text-center'>Manage your existing donation</h3>
+                  {[
+                    `Update your existing donation`,
+                    `Download donation receipts`
+                  ].map(donationSettingOps => (
```

Comment:
It looks like these two buttons both just point to the same place. Should they do different things?

Reply:
This is an intended behaviour, we wanted to let users know at the first glance that they can do different things.